src/storage/ynab.ts

```diff
@@ -0,0 +1,103 @@
+import { YNAB_TOKEN, YNAB_BUDGET_ID, YNAB_ACCOUNTS } from "../config.js";
+import { SaveStats, TransactionRow, TransactionStorage } from "../types.js";
+import { createLogger } from "./../utils/logger.js";
+import { parseISO, format } from "date-fns";
+import * as ynab from "ynab";
```

Comment:
Can you add `"ynab"` to the `cSpell.words` list in `.vscode\settings.json`?

Reply:
done